packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -533,10 +534,7 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
           <Text style={style.subText}>
             {inputLabelText[usage]}
             {usage === PINEntryUsage.ChangeBiometrics && (
-              <Text style={style.parenthesisText}>
-                {` `}
-                {t('PINEnter.ChangeBiometricsInputLabelParenthesis')}
-              </Text>
+              <Text style={style.parenthesisText}>{t('PINEnter.ChangeBiometricsInputLabelParenthesis')}</Text>
```

Comment:
I think this space was originally added by one of the Quebec or Ontario devs, I forget why. Can probably keep it as is for now so the snapshot doesn't break